### ISSUE TITLE
gitlab ci 

### ISSUE THREAD

**Issue by Cc360428** [go-redis/redis#1588]
## go vet $(go list ./... | grep -v /vendor/)
```go
$ go fmt $(go list ./... | grep -v /vendor/)
$ go vet $(go list ./... | grep -v /vendor/)
# *******/vendor/github.com/go-redis/redis/v8
vendor/github.com/go-redis/redis/v8/redis.go:222:13: undefined: errors.Unwrap
ERROR: Job failed: exit code 1
```
## go.mod version
```go
go 1.14
require (
github.com/go-redis/redis/v8 v8.4.4
)
```

**Comment by vmihailenco** [comment]
errors.Unwrap is available since Go 1.13.

**Comment by weirdian2k3** [comment]
@vmihailenco should the `go mod` version be updated in the https://github.com/go-redis/redis/blob/master/go.mod file if it requires 1.13?
According to this: https://golang.org/ref/mod#go-mod-file-go, because the go.mod file is set to 1.11, it's being built with the features available in 1.11

I'm trying to use this on a machine that only has 1.15.6 and I'm receiving the same error

**Comment by vmihailenco** [comment]
@weirdian2k3 not sure, but I've just bumped Go version in go.mod
